[29042022 16:48:23] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 16:48:23] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 16:48:26] log /root/dionaea/src/signals.c:36-warning: sigterm_cb loop 0x7f607ae07720 w 0x55c33378c230 revents 1024
[29042022 16:48:44] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 16:48:44] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 16:49:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:50:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:51:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:52:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:53:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:54:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:55:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:56:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:57:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:58:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 16:59:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:00:47] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:00:51] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:00:51] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:01:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:02:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:03:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:04:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:04:59] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:05:02] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:05:02] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:06:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:07:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:08:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:09:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:09:51] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:09:54] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:09:54] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:10:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:11:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:12:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:13:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:14:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:15:45] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:15:47] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:15:47] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:16:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:16:52] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:16:55] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:16:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:17:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:18:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:19:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:20:55] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:21:30] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:21:34] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:21:34] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:22:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:23:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:23:37] tftp /opt/dionaea/lib/dionaea/python/dionaea/tftp.py:655-warning: Unsupported mode: netascii
[29042022 17:23:37] tftp /opt/dionaea/lib/dionaea/python/dionaea/tftp.py:719-warning: Requested file /opt/dionaea/var/dionaea/roots/tftp/a does not exist.
[29042022 17:23:37] connection /root/dionaea/src/connection_udp.c:249-warning: connection error -1 Bad file descriptor
[29042022 17:23:37] log_json /opt/dionaea/lib/dionaea/python/dionaea/log_json.py:234-warning: no attack data for 172.26.0.2:69
[29042022 17:23:37] log_sqlite /opt/dionaea/lib/dionaea/python/dionaea/logsql.py:746-warning: no attackid for 172.26.0.2:69
[29042022 17:24:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:25:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:26:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:27:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:28:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:29:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:30:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:31:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:32:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:33:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:34:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:35:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:36:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:37:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:38:08] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:38:12] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:38:12] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:38:17] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:83-warning: DATABASE opening information_schema
[29042022 17:38:17] mysqld /opt/dionaea/lib/dionaea/python/dionaea/mysql/mysql.py:86-warning: open db information_schema -> :memory:
[29042022 17:39:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:40:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:41:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:42:10] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:42:57] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 17:43:01] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 17:43:01] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 17:44:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:45:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:46:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:47:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:48:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:49:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:50:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:51:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:52:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:52:10] ftp /opt/dionaea/lib/dionaea/python/dionaea/ftp.py:235-warning: cmd 'b'AUTH''
[29042022 17:53:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:54:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:55:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:56:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:57:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:57:08] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:776-warning: Attempt to register 99fcfec4-5260-101b-bbcb-00aa0021347a failed, TransferSyntax 6cb71c2c-9812-4540-0100-000000000000 is unknown
[29042022 17:57:08] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[29042022 17:57:08] SMB /opt/dionaea/lib/dionaea/python/dionaea/smb/smb.py:776-warning: Attempt to register 99fcfec4-5260-101b-bbcb-00aa0021347a failed, TransferSyntax 6cb71c2c-9812-4540-0100-000000000000 is unknown
[29042022 17:57:08] connection /root/dionaea/src/connection_tcp.c:271-warning: recv failed size -1 recv_size 1 (Connection reset by peer)
[29042022 17:58:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 17:59:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:00:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:01:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:02:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:03:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:04:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:05:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:06:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:07:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:08:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:09:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:10:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:11:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:12:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:13:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:14:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:15:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:16:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:17:05] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:17:52] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:17:52] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 18:17:52] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:37:11] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[29042022 19:37:11] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[29042022 19:37:11] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 19:37:30] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 19:37:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 19:38:30] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:38:55] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[29042022 19:38:55] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[29042022 19:39:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:40:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:41:45] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:43:00] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:43:22] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
[29042022 19:44:07] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[29042022 19:44:07] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[29042022 19:47:01] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:49:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:49:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:49:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:49:46] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[29042022 19:49:46] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[29042022 19:49:46] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[29042022 21:08:53] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[29042022 21:08:53] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[29042022 21:08:53] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[29042022 21:08:53] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[29042022 21:08:53] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
